src/ResourceManager/Network/Commands.Network/help/Get-AzureRmP2SVpnGateway.md

```diff
@@ -0,0 +1,98 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzureRmP2SVpnGateway
+
+## SYNOPSIS
```

Comment:
Fill in the synopsis and description for all these new files.

Reply:
Done.